sphinxcontrib_robotframework.py

```diff
@@ -87,7 +87,7 @@ def run_robot(app, doctree, docname):
     # Skip already run robotframework suites
     checksums_filename = os.path.join(app.doctreedir, ROBOT_PICKLE_FILENAME)
     try:
-        with open(checksums_filename) as fp:
+        with open(checksums_filename, 'wb') as fp:
```

Comment:
This file is for reading, so the mode needs to be `'rb'` (or  `'b'` which is the same thing, and which is what I mentioned in my previous comment).

If you use `'wb'` here, then `fp.read()` in the next line will raise `io.UnsupportedOperation: read`.

```suggestion
        with open(checksums_filename, 'b') as fp:
```